website/pages/docs/builders/amazon/ebs.mdx

```diff
@@ -143,12 +143,22 @@ run:
 <Tab heading="HCL2">
 
 ```hcl
+// To make Packer read these variables from the environment into the var object,
+// set the environment variables to have the same name as the declared
+// variables, with the prefix PKR_VAR_.
+
+// You could also hardcode them into the file, but we recommend against that.
+
+// export PKR_VAR_aws_access_key=$YOURKEY
```

Comment:
Should this be in a partial ?

Reply:
I think it's an over-optimization at the moment, especially since tabs + partials aren't playing nice yet.